pandas/_libs/groupby_helper.pxi.in

```diff
@@ -879,6 +879,51 @@ def group_cummax_{{name}}(ndarray[{{dest_type2}}, ndim=2] out,
 #----------------------------------------------------------------------
 
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def group_any(ndarray[int64_t] out,
+              ndarray values,
+              ndarray[int64_t] labels,
+              bint skipna):
+    """Aggregated boolean values to show if any group element is truthful
+
+    Parameters
+    ----------
+    out : array of int64_t values which this method will write its results to
+    values : array of values to be truth-tested
+    labels : array containing unique label for each group, with its ordering
+        matching up to the corresponding record in `values`
+    skipna : boolean
+        Flag to ignore nan values during truth testing
+
+    Notes
+    -----
+    This method modifies the `out` parameter rather than returning an object.
+    The returned values will either be 0 or 1 (False or True, respectively).
+    """
+    cdef:
+        Py_ssize_t i, N=len(labels)
+        int64_t lab
+        ndarray[int64_t] bool_mask
+        ndarray[uint8_t] isna_mask
+
```

Comment:
so related to some of my comments on Groupby.ffill. I think should be consistent on the mask. IOW there you are passing it in (which I think is actually easier), rather than computing it here. ideally we would have very similar signatures for ffill, shift_indexer, group_any and such. (they are pretty close now).